### ISSUE TITLE
Share the Data

### ISSUE THREAD

**Issue by Team-thedatatribune** [Team-thedatatribune/IPL-Analysis#4]
# Share Prepared Data

We highly appreciate if you can extract certain specific kind of data (from what we already have) viz    
> **Season-wise**  
> **1<sup>st</sup> powerplay-wise** (initial 5 overs) wise etc...

---

The projects has a really wide scope of study, so we intend to provide all such data prepared in advanced that might prove to be helpful in good data science studies!  
  
Apart from that, you are free to share the different type of datasets you find from different valid sources, about the ipl statistics.  
The dataset related to different cricketers or teams or stadiums can also be very helpful.  

If you want to update the datasets, that would also be welcomed!!

**Comment by kishanpython** [comment]
I want to add ipl data for the analysis. Can you assign it to me?

**Comment by Team-thedatatribune** [comment]
> I want to add ipl data for the analysis. Can you assign it to me?

Yeah, make sure that you add a little description about your dataset in a separate markdown file in the [```dataset-defination```](https://github.com/Team-thedatatribune/IPL-Analysis/tree/dataset-defination) branch (_```your-dataset-name.md```_).

Put the dataset in the [```Datasets```](https://github.com/Team-thedatatribune/IPL-Analysis/tree/dataset-defination/Datasets) directory only!

**Comment by kishanpython** [comment]
yeah
